### DISCUSSION TITLE
Search Feature

### DISCUSSION THREAD

**Discussion by nwazuo** [ShahriarKh/twemoji-cheatsheet#7] [Ideas]
Hi ShahriarKh, thanks for you awesome work with this. 

I thought a search feature would be really awesome where we can search by a string. I was trying to look for a specific flag after filtering by the flag group and it was a pain to have to scan the entire list, trying to match the flag with my eyes. 

If you need a hand, I can lend some of my time to working on this feature. We can discuss an approach in this GitHub issue thread and then we can get to work on implementing it.

**Comment by ShahriarKh** [comment]
Great! Feel free to take a look at source and see what you can do.

**Comment by ShahriarKh** [comment]
@nwazuo The search is now live!

**Reply by nwazuo** [reply]
🎉yay! amazing. let me try it out.
